charts/seed-bootstrap/templates/gardener-seed-admission-controller/rbac.yaml

```diff
@@ -0,0 +1,46 @@
+---
+apiVersion: {{ include "rbacversion" . }}
+kind: ClusterRole
+metadata:
+  name: gardener-seed-admission-controller
+  labels:
+    app: gardener
+    role: seed-admission-controller
+rules:
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
+  verbs:
+  - get
+  - list
+- apiGroups:
+  - extensions.gardener.cloud
+  resources:
+  - backupbuckets
+  - backupentries
+  - controlplanes
+  - extensions
+  - infrastructures
+  - networks
+  - operatingsystemconfigs
+  - workers
```

Comment:
I didn't add it because it's not yet merged. But yes, it makes sense to already add it now in order to prevent follow-up effort. It doesn't harm here if the #2035 is not yet merged. Done, thanks.
PS: I also asked to add the protection label to the new CRD here: https://github.com/gardener/gardener/pull/2035#discussion_r395293604